readme_renderer/__main__.py

```diff
@@ -1,19 +1,65 @@
 import argparse
-from readme_renderer.rst import render
+import email
+from readme_renderer.markdown import render as render_md
+from readme_renderer.rst import render as render_rst
+from readme_renderer.txt import render as render_txt
+import pathlib
+from pkg_resources import get_distribution
 import sys
 
 
-if __name__ == '__main__':
+def __main__(args=None):  # noqa: N807
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**suggestion:** ‏instead of disabling the linter here, how about rename the function to be `main()` instead?